proto/dymensionxyz/dymension/rollapp/params.proto

```diff
@@ -24,4 +24,9 @@ message Params {
   uint64 liveness_slash_interval = 5  [(gogoproto.moretags) = "yaml:\"liveness_slash_interval\""];
   // The time (num hub blocks) a sequencer can be down after which he will be jailed rather than slashed
   uint64 liveness_jail_blocks = 6 [(gogoproto.moretags) = "yaml:\"liveness_jail_blocks\""];
+  // app_creation_cost is the cost for registering the App
+  cosmos.base.v1beta1.Coin app_creation_cost = 7 [
```

Comment:
Id suggest naming it to more conventional name, which is `fee` and not `cost`.
so `app_registration_fee`